src/Components/Server/src/Circuits/RemoteNavigationManager.cs

```diff
@@ -67,28 +67,33 @@ public void NotifyLocationChanged(string uri, bool intercepted)
         /// <inheritdoc />
         protected override void NavigateToCore(string uri, bool forceLoad)
         {
-            Log.RequestingNavigation(_logger, uri, forceLoad);
+            NavigateToCore(uri, forceLoad ? NavigationOptions.ForceLoad : NavigationOptions.None);
+        }
+
+        /// <inheritdoc />
+        protected override void NavigateToCore(string uri, NavigationOptions options)
+        {
+            Log.RequestingNavigation(_logger, uri, options);
 
             if (_jsRuntime == null)
             {
                 var absoluteUriString = ToAbsoluteUri(uri).ToString();
                 throw new NavigationException(absoluteUriString);
             }
-
-            _jsRuntime.InvokeAsync<object>(Interop.NavigateTo, uri, forceLoad);
+            _jsRuntime.InvokeAsync<object>(Interop.NavigateTo, uri, (options & NavigationOptions.ForceLoad) !=0, (options & NavigationOptions.Replace) !=0);
```

Comment:
Minor spacing tweak:

```suggestion
            _jsRuntime.InvokeAsync<object>(Interop.NavigateTo, uri, (options & NavigationOptions.ForceLoad) != 0, (options & NavigationOptions.Replace) != 0);
```